R/plots.R

```diff
@@ -358,14 +347,18 @@ plot_param_vs_iter <- function(x, eval_time = NULL) {
   p <-
     ggplot(x, aes(x = .iter, y = value)) +
     geom_point() +
-    xlab("Iteration") +
-    ylab("") +
-    facet_wrap(~name, scales = "free_y")
+    xlab("Iteration")
+
```

Comment:
```suggestion
    xlab("Iteration") + 
    scale_x_continuous(breaks = unique(x$.iter))

```

Reply:
This should be applied to the other time `xlab("Iteration") is used in this file.